includes/documents/class-wcpdf-invoice.php

```diff
@@ -456,6 +459,48 @@ public function get_date_title() {
 		return apply_filters( "wpo_wcpdf_{$this->slug}_date_title", $date_title, $this );
 	}
 
+	/**
+	 * Schedules the invoice yearly reset number with AS
+	 */
+	public function schedule_yearly_reset_number() {
+		// checks AS functions existence
+		if( ! function_exists( 'as_schedule_single_action' ) || ! function_exists( 'as_get_scheduled_actions' ) ) {
+			return;
+		}
+
+		$next_year = strval( intval( current_time( 'Y' ) ) + 1 );
+		$datetime  = new \WC_DateTime( "{$next_year}-01-01 00:00:01", new \DateTimeZone( 'UTC' ) );
+
+		// set local timezone or offset
+		if ( get_option( 'timezone_string' ) ) {
```

Comment:
is there a reason you're checking this option? You're not using it directly and I think `wc_timezone_string()` handles this already?